### DISCUSSION TITLE
Mounting blobfuse filesystem with sytemd service not working

### DISCUSSION THREAD

**Discussion by marbaa** [Azure/azure-storage-fuse#980] [Q&A] [answered]
Hi,

OS: RHEL7.9
Blobfuse: blobfuse-1.4.5-1.el7.x86_64

I would like to make automount of our blobfuse filesystem but don't want to use fstab. I do not wan't to prevent boot to OS if internet connection is not working.

I have bash script for mounting blobfuse, that calls config file. It works fine, I can execute script from anywhere and filesystem is mounted.
```
...
MYNAME="azure"
MYHOME="/root/blobfuse"
blobfuse /mnt/$MYNAME \
        --tmp-path=/tmp/$MYNAME -o allow_other -o attr_timeout=240 -o entry_timeout=240 -o negative_timeout=120 \
        --log-level=LOG_DEBUG \
        --file-cache-timeout-in-seconds=120 \
        --config-file=$MYHOME/$MYNAME.cfg
```

However if I call this script from systemd.service file filesystem is not mounted and it ends with message ```Transport endpoint is not connected```

```
[root]# ls /mnt/azure/
ls: cannot access /mnt/azure/: Transport endpoint is not connected
[root@node2.papenburg.de.warp system]# ls -l /mnt
ls: cannot access /mnt/azure: Transport endpoint is not connected
total 8
d????????? ? ?    ?       ?            ? azure
```

Systemd service file:
```
# cat /etc/systemd/system/azure-mount.service
[Unit]
Description=Mount Azure filesystem
After=default.target

[Service]
Type=oneshot
ExecStart=/root/blobfuse/azure.sh

[Install]
WantedBy=default.target
```

**Comment by bhansalivikas** [comment, marked as answer]
Sample config file to create a service based on blobfuse is [here](https://github.com/Azure/azure-storage-fuse/blob/master/systemd/with-config-file/blobfuse.service)

**Reply by marbaa** [reply]
Thanks. Will try and post result later.

**Comment by mholtz16** [comment]
I run my mount script via Cron with an entry like this:

`@reboot /usr/bin/mount_fuse.sh`
